### ISSUE TITLE
ATLAS: Token-level data

### ISSUE THREAD

**Issue by jacobwegner** [jhu-digital-manuscripts/AnIOp#49]
To support viewing morphological and interlinear display functionality

**Comment by jacobwegner** [comment]
_Week of February 17th_

We've been working on modeling token-level data in ATLAS (see [scaife-viewer/sv-mini-atlas#7])(https://github.com/scaife-viewer/sv-mini-atlas/pull/7); hopefully we'll be able to ingest some of the token-level Homer data provided by Greg Crane early next week.

Supporting the data modeling and ingestion, we've also done some work to optimize the ingestion pipeline.

**Comment by jacobwegner** [comment]
_Week of February 24th_

Continued work on token-level data ingestion, now queryable via the https://mini-stack-a-spike-toke-t7jtur.herokuapp.com/graphql endpoint:

```
{
  textParts(urn_Startswith: "urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.", first: 20) {
    edges {
      node {
        ref
        textContent
        tokens {
          edges {
            node {
              value
              idx
              lemma
              tag
              partOfSpeech
              case
              mood
            }
          }
        }
      }
    }
  }
}
```

as cURL:
```
curl 'https://mini-stack-a-spike-toke-t7jtur.herokuapp.com/graphql/#' \
    -XPOST \
    -H 'Content-Type: application/json' \
    --data-binary '{"query":"{\n  textParts(urn_Startswith: \"urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.\", first: 20) {\n    edges {\n      node {\n        ref\n        textContent\n        tokens {\n          edges {\n            node {\n              value\n              idx\n              lemma\n              tag\n              partOfSpeech\n              case\n              mood\n            }\n          }\n        }\n      }\n    }\n  }\n}","variables":null}'
```

![image](https://user-images.githubusercontent.com/629062/75418224-8ed9e600-58f8-11ea-9812-649fcc3c0bc2.png)

We've also started some prototyping to support a couple of other  `Epic: SV-UC3` tickets:

- As a reader, I can view an interlinear display of a transcription #59
- As a reader, I can view morphological data for a transcription #60

which can be previewed via
https://tokens-demo--sv-mini.netlify.com/reader?urn=urn%3Acts%3AgreekLit%3Atlg0012.tlg001.perseus-grc2%3A1.1-1.7

![image](https://user-images.githubusercontent.com/629062/75418453-23444880-58f9-11ea-9ef0-63957c087e14.png)

(I plan on adding #59 and #60 to the project board at a later time, as we're still more focused on the data rather than full implementation of those tickets)

**Comment by jacobwegner** [comment]
_Week of March 2nd_

@jacobwegner was out sick this week.  Our team wrote some integration tests to wrap up the ingestion pipeline work started week of February 17th.

**Comment by jacobwegner** [comment]
Resolved by [explorehomer-atlas/pull/38](https://github.com/scaife-viewer/explorehomer-atlas/pull/38)

Information on querying the morphosyntactic token annotations (lemma, part of speech tag, etc) via GraphQL has been documented within [AniOp ATLAS Documentation](https://docs.google.com/document/d/1XB0vYGxY1M420lmpJhLe2M3IfzcOQwBYFiwKGaDQ9ug/edit).
